db/migrate/20160106040041_alter_loans.rb

```diff
@@ -0,0 +1,12 @@
+class AlterLoans < ActiveRecord::Migration
+  def change
+    add_column :loans, :name, :string
```

Comment:
Okay.  I see now that I was confused about what db:setup does versus db:create db:migrate db:seed.  So I wasn't properly regression testing the full set of migrate scripts.  Looks like that was the only problem though.
One more thought.  It seems that it would be useful in general to put system data population in both seeds.rb and migrate scripts.  That way developers don't need to worry about knowing when to reapply the db:seed step.